src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AddTransparentDataEncryptionCertificateTests.ps1

```diff
@@ -0,0 +1,270 @@
+$password = "AE"
+$location = "WestCentralUS"
+$exceptionHeader = "##### Following exception occurred causing test failure #####"
+$mangedInstanceRg = "MlAndzic_RG"
+$managedInstanceName = "midemoinstance"
+
+function Test-AddTdeCertificateForSqlServerDefaultParameterSetNoPassword
+{
+	# Setup
+	$rg = Create-ResourceGroupForTest
+	$server = Create-ServerForTest $rg $location
+	try
+	{
+		Add-AzureRmSqlServerTransparentDataEncryptionCertificate `
```

Comment:
@nivimsft for all of the tests below, it doesn't look like we're validating the result of the cmdlet invocation, but just making sure the cmdlet doesn't fail. We need to add some asserts after the cmdlet to verify the execution of the cmdlet.

Reply:
Yes, the cmdlet under test adds the TDE certificate to the SQL Server. There is not get method to retrieve the certificate so, we cannot validate to verify the state after execution using asserts. However, the API will throw an exception for any failures, so just ensuring that the execution succeeds is enough to establish that the execution of cmdlet succeeded